flow-defs/text-field-base.js.flow

```diff
@@ -52,7 +52,6 @@ export type CommonFormFieldProps = {
   value?: string,
   defaultValue?: string,
   onChange?: (event: SyntheticEvent<HTMLInputElement>) => void,
-  readOnly?: boolean,
```

Comment:
I think this has removed the `readOnly` property from the `TextField` component props:
![image](https://user-images.githubusercontent.com/4521712/201078400-c35d60d9-3f76-4dca-80d2-20c8dff1788e.png)

This is because `TextField` has `TextFieldProps` that extends from  `CommonFormFieldProps`, and none of them has the `readOnly` prop since this change.
@atabel @pladaria @yceballost

Reply:
https://jira.tid.es/browse/WEB-701